lib/rubycritic/source_control_systems/git.rb

```diff
@@ -30,7 +30,7 @@ def self.to_s
       end
 
       def churn
-        @churn ||= Churn.new(churn_after: Config.churn_after)
+        @churn ||= Churn.new(churn_after: Config.churn_after, paths: Config.paths)
```

Comment:
@Fito I'm not sure about this. How does this work with the `.rubycritic.yml` configuration file?

Reply:
@etagwerker This change works both for the `paths` options defined in `.rubycritic.yml` and for paths provided via the command line.

The `RubyCritic::Cli::Options#to_h` method merges all options provided via the command line and via the `.rubycritic.yml` configuration file. [here](https://github.com/whitesmith/rubycritic/blob/main/lib/rubycritic/cli/options.rb#L23)

`RubyCritic::Cli::Application` instantiates an `RubyCritic::Cli::Options` object and calls `to_h` on it before passing it into `RubyCritic::CommandFactory.create` to execute the reporter [here](https://github.com/whitesmith/rubycritic/blob/main/lib/rubycritic/cli/application.rb#L21).

The `RubyCritic::CommandFactory` sets the options hash on the global config object via `RubyCritic::Config.set(options)` [here](https://github.com/whitesmith/rubycritic/blob/main/lib/rubycritic/command_factory.rb#L10).

This means `Config.paths` will contain the paths provided to RubyCritic via either CLI or Config File options.